Backend/dev/migrations/provider-dashboard/0022-fleet-owner-access.sql

```diff
@@ -3,4 +3,8 @@ insert into atlas_bpp_dashboard.access_matrix (id, role_id, api_entity, user_acc
 insert into atlas_bpp_dashboard.access_matrix (id, role_id, api_entity, user_access_type, created_at, updated_at, user_action_type) (select atlas_bpp_dashboard.uuid_generate_v4() as id,id as role_id,'FLEET' as api_entity, 'USER_FULL_ACCESS' as user_access_type, now() as created_at, now() as updated_at, 'GET_ALL_DRIVERS_FOR_FLEET' as user_action_type from atlas_bpp_dashboard.role) on conflict do nothing;
 insert into atlas_bpp_dashboard.access_matrix (id, role_id, api_entity, user_access_type, created_at, updated_at, user_action_type) (select atlas_bpp_dashboard.uuid_generate_v4() as id,id as role_id,'FLEET' as api_entity, 'USER_FULL_ACCESS' as user_access_type, now() as created_at, now() as updated_at, 'GET_ALL_VEHICLE_FOR_FLEET' as user_action_type from atlas_bpp_dashboard.role) on conflict do nothing;
 insert into atlas_bpp_dashboard.access_matrix (id, role_id, api_entity, user_access_type, created_at, updated_at, user_action_type) (select atlas_bpp_dashboard.uuid_generate_v4() as id,id as role_id,'FLEET' as api_entity, 'USER_FULL_ACCESS' as user_access_type, now() as created_at, now() as updated_at, 'FLEET_REMOVE_VEHICLE' as user_action_type from atlas_bpp_dashboard.role) on conflict do nothing;
-insert into atlas_bpp_dashboard.access_matrix (id, role_id, api_entity, user_access_type, created_at, updated_at, user_action_type) (select atlas_bpp_dashboard.uuid_generate_v4() as id,id as role_id,'FLEET' as api_entity, 'USER_FULL_ACCESS' as user_access_type, now() as created_at, now() as updated_at, 'FLEET_REMOVE_DRIVER' as user_action_type from atlas_bpp_dashboard.role) on conflict do nothing;
\ No newline at end of file
+insert into atlas_bpp_dashboard.access_matrix (id, role_id, api_entity, user_access_type, created_at, updated_at, user_action_type) (select atlas_bpp_dashboard.uuid_generate_v4() as id,id as role_id,'FLEET' as api_entity, 'USER_FULL_ACCESS' as user_access_type, now() as created_at, now() as updated_at, 'FLEET_REMOVE_DRIVER' as user_action_type from atlas_bpp_dashboard.role) on conflict do nothing;
```

Comment:
why FULL_ACCESS to everyone 😓

Reply:
ohh i added this is local by mistake it came here i will change.